www/js/lib/uiUtil.js

```diff
@@ -406,9 +406,14 @@ define(rqDef, function(settingsStore) {
      * Removes the animation effect between various sections
      */
     function removeAnimationClasses() {
-        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
-        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
-        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+        document.getElementById('about').classList.remove('slideIn_L');
+        document.getElementById('about').classList.remove('slideOut_R');
+        document.getElementById('configuration').classList.remove('slideIn_L');
+        document.getElementById('configuration').classList.remove('slideIn_R');
+        document.getElementById('configuration').classList.remove('slideOut_L');
+        document.getElementById('configuration').classList.remove('slideOut_R');
```

Comment:
Given that `document.getElementById('configuration')` appears four times, it would be better to define it once as a variable (`let configuration = ...`) and then just operate on the variable.

Reply:
Alright, I shall do this.